contracts/Makefile

```diff
@@ -20,14 +20,14 @@ build: version ## Build contracts.
 
 
 CONTRACTS := OmniPortal FeeOracleV1 OmniAVS OmniStake Create3 TransparentUpgradeableProxy \
-			DelegationManager StrategyManager StrategyBase AVSDirectory \
+			DelegationManager StrategyManager StrategyBase AVSDirectory XRegistry XRegistryReplica PortalRegistry \
 			src/deploy/ProxyAdmin.sol:ProxyAdmin \
 			test/avs/common/MockERC20.sol:MockERC20
 
 .PHONY: bindings
 bindings: check-abigen-version build ## Generate golang contract bindings.
 	./bindings/scripts/gen.sh $(CONTRACTS)
-	./bindings/scripts/genmore.sh OmniStake Create3 TransparentUpgradeableProxy src/deploy/ProxyAdmin.sol:ProxyAdmin
+	./bindings/scripts/genmore.sh OmniStake Create3 TransparentUpgradeableProxy src/deploy/ProxyAdmin.sol:ProxyAdmin XRegistry PortalRegistry
```

Comment:
Do we need the `XRegistryReplica` here as well?

Reply:
No because it's not predeployed. We generate regular bindings for it